docs/api/README.md

```diff
@@ -175,37 +185,139 @@ status:
+| Data type of Result                                           | `summary.type`                       | The `type` of a Result is a child of `summary``.                     |
+| Name of the first step of the tasks of a run from its status. | `data.status.taskSpec.steps[0].name` | The JSON path is `data -> status -> taskSpec -> steps -> 0 -> name`. |
+
+#### Using Operators
+
+Now that we can access the fields, you can create a filter using operators. Here
+is a list of operators that can be used in CEL expressions:
+
+| Operator                | Description          | Example                                                                                        |
+| ----------------------- | -------------------- | ---------------------------------------------------------------------------------------------- |
+| `==`                    | Equal to             | `data_type == "tekton.dev/v1beta1.TaskRun"`                                                    |
+| `!=`                    | Not equal to         | `summary.status != "SUCCESS"`                                                                  |
+| `IN`                    | In a list            | `data.status.podName in ['hello', 'hello-hello-pod']`                                          |
+| `!`                     | Negation             | `!(data.status.podName in ['hello', 'hello-hello-pod'])`                                       |
+| `&&`                    | Logical AND          | `data_type == "tekton.dev/v1beta1.TaskRun" && name.startsWith("foo/results/bar")`              |
+| `\|\|`                  | Logical OR           | `data_type == "tekton.dev/v1beta1.TaskRun" \|\| data_type == "tekton.dev/v1beta1.PipelineRun"` |
+| `+`, `-`, `*`, `/`, `%` | Arithmetic operators | `data.status.completionTime - data.status.startTime > timestamp("2021-02-02T22:37:32Z")`       |
+| `>`, `>=`, `<`, `<=`    | Comparison operators | `data.status.completionTime > data.status.startTime`                                           |
+
+#### Using Functions
+
+There are many functions that can be used in CEL expressions. Here is a
+list of functions that can be used in CEL expressions. The string in the function
+argument shows the expected type of the argument:
+
+| Functions                                             | Description                                                       | Example                                                        |
+| ----------------------------------------------------- | ----------------------------------------------------------------- | -------------------------------------------------------------- |
+| `startsWith('string')`                                | Checks if a string starts with a prefix                           | `name.startsWith("foo")`                                       |
```

Comment:
To be consistent, you should have `data.metadata.name.startswith("foo")`,

Reply:
Umm I wanted to show the function on `name` field of record, but I will change it.